lib/cartopy/feature.py

```diff
@@ -377,13 +435,13 @@ def intersecting_geometries(self, extent):
 
 LAND = NaturalEarthFeature('physical', 'land', '110m',
                            edgecolor='face',
-                           facecolor=COLORS['land'], zorder=-1)
```

Comment:
Why change this?

Reply:
The `zorder` was added a couple of days ago for `LAND` and `OCEAN`. It should remain.